go.mod

```
@@ -5,6 +5,7 @@ go 1.17
 replace (
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
 	github.com/graymeta/stow => github.com/kastenhq/stow v0.2.6-kasten.1.0.20220719172801-c11a0e5f2e09
+	github.com/kastenhq/kopia => github.com/kastenhq/kopia v0.0.0-20220726032646-b45407588e7b
```

```suggestion
	github.com/kopia/kopia => github.com/kastenhq/kopia v0.0.0-20220726032646-b45407588e7b
```
We should replace `kopia/kopia` with `kastenhq/kopia`. Also, once you make this change, please run `go mod tidy -compat=1.17` again

---

Probably you should run without -compat. When I tried just now it broke a pipeline step: https://github.com/kanisterio/kanister/runs/7537284808?check_suite_focus=true  -- switching back to `go mod tidy` instead

---

Yeah, I faced the similar issue, I also switched back to `go mod tidy`.